src/stories/Digit.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import { Meta, StoryObj } from "@storybook/react";
 
 import { Digit } from "../components/Digit";
+import CharToDigit from "../utils/charToDigit";
```

Comment:
Here, the import should be `chatToDigit`

Reply:
I don't quite get it? Do I have a typo? Do you😅?

Reply:
Yes, it's a typo. Since `utils/charToDigit.ts` exports `charToDigit` by default, you can rename it when you import it, that's why this code works anyways. But it should be `charToDigit` since it is the array that you want and not the type. Don't worry, I'll merge the PR and then fix it.